packages/blocks/src/api/raw-handling/test/utils.js

```diff
@@ -109,6 +109,12 @@ describe( 'removeInvalidHTML', () => {
 		expect( removeInvalidHTML( input, schema ) ).toBe( output );
 	} );
 
+	it( 'should remove id attributes', () => {
+		const input = '<p id="foo">test</p>';
+		const output = '<p>test</p>';
+		expect( removeInvalidHTML( input, schema ) ).toBe( output );
+	} );
```

Comment:
Is it possible to add a unit tests for headings (where we keep the attribute?)

Reply:
Not without moving things around, or copy/pasting code. Testing that functionality requires access to `getRawTransformations()`, which currently isn't exported.